pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -210,14 +244,20 @@ async def run(self):
 
         # bugs should be attached after builds to validate tracker bugs against builds
         _LOGGER.info("Sweep bugs into the the advisories...")
-        if assembly_type in (AssemblyTypes.PREVIEW, AssemblyTypes.CANDIDATE):
-            self.sweep_bugs(permissive=True)
-            _LOGGER.info("Skipping flaw processing during pre-releases")
-        else:
-            self.sweep_bugs(permissive=False)
-            _LOGGER.info("Processing attached Security Trackers")
+
+        if self.advance_release:
+            self.sweep_bugs(permissive=False, advance_release=True)
```

Comment:
we need to move dependencies builds to advance advisory before sweep bug

Reply:
looks like bug sweep happens before we sweep/move builds